packaging/opensuse-15.1

```diff
@@ -1 +0,0 @@
-opensuse
```

Comment:
I notice we still have packaging/opensuse-15.0, is that needed still since we are dropping 15.1 ?

Reply:
Ah the git diff shows it as just moving from opensuse-15.0 to opensuse-15.3 sorry I was just confused